src/main/java/rx/Observer.java

```diff
@@ -34,7 +34,7 @@
      * <p>
      * The {@link Observable} will not call this method if it calls {@link #onError}.
      */
-    public abstract void onCompleted();
+    void onCompleted();
```

Comment:
This is a breaking change. E.g., RxScala has a Scala Observer wrapper that calling these methods. Removing `public abstract` will break them.